sycl/test-e2e/Graph/Explicit/debug_print_graph.cpp

```diff
@@ -0,0 +1,34 @@
+// REQUIRES: level_zero, gpu
```

Comment:
Can this test run on  CUDA? Same comment for other new tests

Reply:
Yes, it works. I added CUDA for these tests.